cli/jobs/automl-standalone-jobs/cli-automl-image-classification-multilabel-task-fridge-items/prepare_data.py

```diff
@@ -10,13 +10,42 @@
 from azure.ai.ml.constants import AssetTypes
```

Comment:
Should we consolidate the prepare_data in a single file and reuse across notebooks? It has a lot of duplication now 

Maybe check with Phani and Riti if there is a strong reason for this duplication.

Reply:
I think the reason was that - if we have prepare_data.py here, the user can copy the folder anywhere and follow the README.md instructions and it would work. Otherwise, this file specifically needs to be copied from the correct path.

Reply:
+1 to @PhaniShekhar... I think last time we thought about consolidating, point was raised that all example folders should be self-contained...